cmd/generate-plugin-overview/go.mod

```diff
@@ -0,0 +1,9 @@
+module sigs.k8s.io/krew/cmd/generate-plugin-overview
+
+go 1.12
+
+require (
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/pkg/errors v0.8.1 // indirect
+	sigs.k8s.io/krew v0.2.2-0.20190724210953-8867a94acd6d
```

Comment:
funny enough this doesn't build from root

```
$ go build ./cmd/generate-plugin-overview

go: finding sigs.k8s.io/krew/cmd/generate-plugin-overview latest
go: finding sigs.k8s.io/krew/cmd latest
can't load package: package sigs.k8s.io/krew/cmd/generate-plugin-overview: unknown import path "sigs.k8s.io/krew/cmd/generate-plugin-overview": cannot find module providing package sigs.k8s.io/krew/cmd/generate-plugin-overview
```

but if you `./cmd/generate-plugin-overview` then `go build .` it works haha.